toolkits/springer/packages/springer-hero/scss/50-components/springer-hero.scss

```diff
@@ -0,0 +1,42 @@
+.c-hero {
+	display: flex;
+	flex-wrap: wrap;
+	background-color: $hero--background-color;
+	color: #fff;
```

Comment:
The default brand context has `$context--foreground-light` as `#fff` but that gets overridden if used here with tbe Springer brand variable.

Just making a note - I think the `shared` colours for the default brand context could do with variables for black and white (and a greyscale) so we don't have to write hex codes.